GitUI/UserControls/RevisionGrid.cs

```diff
@@ -2694,26 +2688,32 @@ private void UpdateGraph(GitRevision rev)
 
         private void CheckUncommitedChanged(string filtredCurrentCheckout)
         {
-            if (UnstagedChanges.Result)
+            var staged = ChangedFiles.Result.Count(item => item.IsStaged);
+            var unstaged = ChangedFiles.Result.Count() - staged;
```

Comment:
referencing `ChangedFiles.Result` twice doesn't feel right
perhaps use a temporary variable

Reply:
Since ChangedFiles is a variable now, it should be fine.